index.html

```diff
@@ -585,8 +591,65 @@ <h1>Introduction</h1>
     <a href="#json-ld-ctx-usage"></a> and the documentation under the namespace IRIs, e.g.,
     <a href="https://www.w3.org/2019/wot/td">https://www.w3.org/2019/wot/td</a>).
 </p>
+</section>
+<section id="introduction-tm">
+    <h2>Thing Model</h2>
+    
+    <p>One of the main intentions of a <a>Thing Description</a> is to provide a <a>Consumer</a> 
+        with all the details necessary to successfully interact with a <a>Thing</a>.  
+        In some IoT application scenarios, a full detailed <a>Thing Description</a>, e.g., with 
+        communication metadata is not necessary (e.g., IoT ecosystems implicitly handle communication separately),
+        or may not be available because a new entity has not yet been deployed (e.g., IP address is not yet known). 
+        Sometime also a kind of class definition is required that forces capability definitions that shall be 
+        available for all created instances (e.g., large-scale production of new devices).
+    </p> 
+        
+    <p>Addressing such or other scenarios the <a>Thing Model</a> can be used that mainly provides the data model 
+        definitions within <a>Things</a>' <a>Properties</a>, <a>Actions</a>, and/or <a>Events</a> and can potential 
```

Comment:
typo: within a Things'

can be potentially used

Reply:
done